### DISCUSSION TITLE
More information about the Sensors + Drivers

### DISCUSSION THREAD

**Discussion by Mahan3340** [bdd100k/bdd100k#182] [Q&A]
Could you provide more information about the sensors, particularly the IMU unit? 
1. How the IMU sensor is positioned? The coordinate system of the sensor is not specified. It seems the x-axis points to gravity.
2. What is the sampling rate of the IMU sensor?
3. Is the IMU a separate sensor installed on the car or smartphone(s) are used. If smartphones, please specific the models for each part of the data, and if a separate IMU sensor, please specify the model as well.

Also, some information about the driver(s) that collected the dataset. In particular,

1. How many drivers collected the data? Is it possible to add a driver label to each segment/record using a random Id? In this way, the data could also be used for driving style recognition or driver identification.

Thanks in Advance.

**Comment by fyu** [comment]
The IMU sensors are from iPhone 5 directly. You may have to dig into the iPhone document for the details. However, we provide code to use the locations information to position the videos on google map.

We have about 50K drivers. As the video name has two ids separated by a dash, the first part is the driver id.

**Reply by Mahan3340** [reply]
Thanks for your response. I’m trying to know the position of the phone  wrt to the car. So how do you placed the phone inside the car? Like the the photo below? ![5F24D889-C2D5-4E64-BB5C-D6DA8F7CD503](https://user-images.githubusercontent.com/21075867/133360061-93beeb9f-58c0-4b2a-9b64-b240e0231be0.jpeg)
